R/run_stress_test.R

```diff
@@ -26,7 +26,7 @@
 #'   excluded_companies.csv shall be excluded.
 #' @return NULL
 #' @export
-run_stress_test <- function(asset_type,
+run_stress_test <- function(asset_type = sort(asset_types_lookup),
```

Comment:
This could be explicitely `c("bonds", "equity", "loans")`, but hiding those types behind an object seems a bit more maintenable.